app/views/searches/show.html.erb

```diff
@@ -0,0 +1,52 @@
+<h1>Search</h1>
+
+<h2>Новый поиск</h2>
+<%= form_tag search_path, method: :get do %>
+  <%= label_tag :starting %>
+  <%= select_tag :starting, options_for_select(station_list) %>
+  <%= label_tag :destination %>
+  <%= select_tag :destination, options_for_select(station_list) %>
+  <%= submit_tag "Search" %>
+<% end %>
+
+<% if @data %>
```

Comment:
`if @data`
`if @data.present?`

Reply:
Something like `@data` not `nill` and not  empty.